osgar/lib/serialize.py

```diff
@@ -0,0 +1,20 @@
+"""
+  serialization of messages
+"""
+from ast import literal_eval
+
+
+def serialize(data):
+    try:
+        bytes_data = data.tobytes()
```

Comment:
This line used to be needed for `numpy`, right? I am just making sure I understand the code right.